README.md

```diff
@@ -95,3 +95,14 @@ The example compose file creates 3x containers:
 * A PHP7.3 version of the openeyes container (on port 7777)
 
 This demonstrates how 2 instances can connect to the same database and use the same code files. It also allows dual PHP version testing, while we migrate to PHP7.
+
+# Adding key for encryption/decryption
+The key to be used for encryption and decryption of the email account password needs to be stored in the key file under [web folder](https://github.com/openeyes/Docker/tree/master/web).
+
+To generate a key having 64 characters in hex format, execute the following command in terminal -
+
+`openssl rand -hex 32`
```

Comment:
Can we automatically create a key in init.sh if one does not already exist?

Reply:
We could, but we want to keep it as a docker secret, which means that it has to exist before the container runs. Especially once we start running as a non-root user.